pkg/ccl/crosscluster/physical/stream_ingestion_dist.go

```diff
@@ -64,7 +65,8 @@ func startDistIngestion(
 	// Start from the last checkpoint if it exists.
 	var heartbeatTimestamp hlc.Timestamp
 	if !replicatedTime.IsEmpty() {
```

Comment:
you should be able to get rid of this if else condition here, since the initialScanTimestamp is the ReplicationStartTime